app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -1,3 +1,7 @@
+<% if @user_deleted %>
+  <%= govuk_panel title: "User Deleted", body: nil %>
```

Comment:
This needs to be a notification banner. Check with @callumacrae

Reply:
@callumacrae make sure you've got `role=alert` is implemented as well.

Reply:
Added flash notice